website/django/data/migrations/0007_versioned_data.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-08-24 16:53
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data', '0006_versioned_data'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='variant',
+            name='Data_Release',
+            # Get data_release_id_seq.start_value instead of assuming 1 ?
```

Comment:
Following up on this, it looks like this migration will fail. This is the error I received when running the migration locally:
```
(brca) Zacharys-MacBook-Air:django zackfischmann$ python manage.py migrate
/Users/zackfischmann/UCSC/brca-exchange/website/django/brca/urls.py:29: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got django.views.static.serve). Pass the callable instead.
  {'document_root': settings.MEDIA_ROOT, 'show_indexes': True})

/Users/zackfischmann/UCSC/brca-exchange/website/django/brca/urls.py:29: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  {'document_root': settings.MEDIA_ROOT, 'show_indexes': True})

/Users/zackfischmann/UCSC/brca-exchange/website/django/brca/urls.py:34: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got django.views.static.serve). Pass the callable instead.
  {'document_root': settings.STATIC_ROOT, 'show_indexes': True})

/Users/zackfischmann/UCSC/brca-exchange/website/django/brca/urls.py:34: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  {'document_root': settings.STATIC_ROOT, 'show_indexes': True})

Operations to perform:
  Apply all migrations: authtoken, users, sessions, admin, sites, auth, contenttypes, data
Running migrations:
  Rendering model states... DONE
  Applying data.0007_versioned_data...Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/core/management/__init__.py", line 345, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/core/management/base.py", line 348, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/core/management/base.py", line 399, in execute
    output = self.handle(*args, **options)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/core/management/commands/migrate.py", line 200, in handle
    executor.migrate(targets, plan, fake=fake, fake_initial=fake_initial)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/db/migrations/executor.py", line 92, in migrate
    self._migrate_all_forwards(plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/db/migrations/executor.py", line 121, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/db/migrations/executor.py", line 198, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/db/backends/base/schema.py", line 90, in __exit__
    self.execute(sql)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/db/backends/base/schema.py", line 110, in execute
    cursor.execute(sql, params)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/db/utils.py", line 95, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/Users/zackfischmann/.virtualenvs/brca/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
django.db.utils.IntegrityError: insert or update on table "variant" violates foreign key constraint "variant_Data_Release_id_34907b0c_fk_data_release_id"
DETAIL:  Key (Data_Release_id)=(1) is not present in table "data_release".
```